specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/examples/FrontdoorCreate.json

```diff
@@ -66,20 +66,24 @@
             "properties": {
               "backends": [
                 {
+                  "type": "Public",
+                  "location": "westus",
```

Comment:
>"location": "westus", [](start = 18, length = 21)

Isnt location valid only for private backend?

Reply:
required for private, but allowed for public as well